ci-operator/config/openshift/velero/openshift-velero-oadp-1.0.yaml

```
@@ -0,0 +1,44 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
```

```suggestion
    tag: golang-1.16
```
to accomodate https://github.com/openshift/velero/pull/249